cfme/intelligence/chargeback/rates.py

```diff
@@ -57,6 +57,9 @@ class fields(ParametrizedView):  # noqa
         PARAMETERS = ('name',)
         ROOT = ParametrizedLocator('.//tr[./td[contains(normalize-space(.), {name|quote})]]')
 
+        def __locator__(self):
+            return self.browser.element(self.ROOT, parent=self.parent)
```

Comment:
Try putting `INDIRECT = True` in the fields class and remove the locator. That may help.